src/emailLogin/README.md

```diff
@@ -43,6 +43,8 @@ This adds the new site member entries and also outputs texts file in `/<repo nam
 
 6. Fill in the appropriate form (prod: https://form.gov.sg/6513ae794fec5f0012b7f8ad) to clone the repo on our EFS.
```

Comment:
trouble your to change to this
https://form.gov.sg/admin/form/653b01a3ec066e001138f1b1